tests/unit/db/database_tests.py

```diff
@@ -601,41 +602,28 @@ def test_view_clean_up(self):
         """
         self.assertEqual(self.db.view_cleanup(), {'ok': True})
 
-    def test_retrieve_changes(self):
+    def test_changes_feed_call(self):
         """
-        Test the retrieval of changes to documents via feed
+        Test that changes() method call constructs and returns a Feed object
         """
-        i = 0
-        doc = self.db.create_document(
-            {'_id': 'julia{0:03d}'.format(i), 'name': 'julia', 'age': i})
-        feed = self.db.changes(feed='continuous')
-        for change in feed:
-            self.assertEqual(change['id'], doc['_id'])
-            i += 1
-            doc = self.db.create_document(
-                {'_id': 'julia{0:03d}'.format(i), 'name': 'julia', 'age': i})
-            if i == 10:
-                feed.stop()
-        self.assertEqual(i, 10)
+        changes = self.db.changes(limit=100)
+        self.assertIsInstance(changes, Feed)
```

Comment:
Again, I think `assertIs(type(changes), Feed)`, is a slightly stronger assertion.